src/database/database.ts

```diff
@@ -107,6 +109,7 @@ const options: DataSourceOptions = {
     PayoutRequestPdf1720610649657,
     DatabaseRbac1720624912620,
     TransfersVat1721916495084,
+    WriteOffs1722004753128,
```

Comment:
Wil TypeORM dat migrations in de juiste volgorde staan? Indien ja, dan moet deze achteraan de lijst